aimlflow/watcher.py

```diff
@@ -82,6 +86,9 @@ def _process_single_run(self, aim_run, mlflow_run):
     def _process_runs(self):
         watch_started_time = time.time()
 
+        # refresh experiments list
+        self._experiments = get_mlflow_experiments(self, self._experiment)
```

Comment:
It seems that there should be `self._client` instead of `self`:
```suggestion
        self._experiments = get_mlflow_experiments(self._client, self._experiment)
```

Otherwise:
```
  File "---/aimlflow/watcher.py", line 90, in _process_runs
    self._experiments = get_mlflow_experiments(self, self._experiment)
  File "---/aimlflow/utils.py", line 71, in get_mlflow_experiments
    experiments = client.search_experiments()
AttributeError: 'MLFlowWatcher' object has no attribute 'search_experiments'
```

The other parts of `get_mlflow_experiments` seem to also expect a different set of methods provided by `client` compared to those provided by `MLFlowWatcher`.

Reply:
oh yeah, thanks a lot for noticing, will update